packages/ibm-products/src/components/SteppedAnimatedMedia/SteppedAnimatedMedia.mdx

```diff
@@ -0,0 +1,38 @@
+import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
+import {
+  getStoryId,
+  CodesandboxLink,
+} from '../../global/js/utils/story-helper';
+import { SteppedAnimatedMedia } from '.';
+
+# SteppedAnimatedMedia
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Example usage](#example-usage)
+- [Component API](#component-api)
+
+## Overview
+
+<!-- TODO: Overview. -->
+
+This component is for the internal use of other Novice to Pro components.
```

Comment:
If you want to keep consistent with what @paul-balchin-ibm had in the Carousel mdx file.

https://github.com/carbon-design-system/ibm-products/pull/3053/commits/df1bfa6d3a7e9a2eae2aa9c1ebc0d7234c05feb5#diff-a5b061b297a9f66494ded70a1b37fcc438507e7c897137e33201d18eb3d71a05R17-R18